Cloud Pak for Data/WML/notebooks/model risk management/OpenScale MRM Cloud pak for Data.ipynb

```diff
@@ -157,7 +158,7 @@
    "source": [
     "WML_CREDENTIALS = WOS_CREDENTIALS.copy()\n",
     "WML_CREDENTIALS['instance_id']='openshift'\n",
-    "WML_CREDENTIALS['version']='3.5' #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"  \n",
+    "WML_CREDENTIALS['version']='4.5' #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"  \n",
```

Comment:
This comment needs to change, since we are specifying 4.5 instead of 3.5 by default